bin/docker/raphtory/DockerfileV2

```diff
@@ -0,0 +1,38 @@
+ARG VERSION
+FROM hseeberger/scala-sbt:eclipse-temurin-11.0.14.1_1.6.2_2.13.8 as sbt-builder
+COPY core core/
```

Comment:
In the distributed build using the previous docker file, we pass raphtory core code in as a jar. This would be useful as a development tool however we need to make the same changes for python to the other dockerfile while leaving the scala code to get copied in by jar